orchestration/hca_orchestration/pipelines/cut_snapshot.py

```diff
@@ -151,13 +151,14 @@ def message_for_snapshot_done(context: HookContext) -> None:
     )
     data_repo_project = snapshot_details.data_project
 
-    context.resources.slack.send_message(blocks=key_value_slack_blocks("HCA Snapshot Complete", {
-        "Snapshot name": context.resources.snapshot_config.snapshot_name,
-        "Dataset Google Project ID": context.resources.hca_manage_config.google_project_name,
-        "Source Dataset": context.resources.snapshot_config.dataset_name,
-        "Snapshot Google Data Project ID": data_repo_project,
-        "Dagit link": f'<{context.resources.dagit_config.run_url(context.run_id)}|View in Dagit>'
-    }))
+    slack_msg_text = ("HCA Snapshot Complete"
+                      + f"\nSnapshot name: {context.resources.snapshot_config.snapshot_name}"
```

Comment:
I think this is fine; alternatively you could do:

```suggestion
lines = [
              "msg msg msg",
              "other msg other msg",
               "etc."
]
slack_msg_text = "\n".join(lines) 
```

i.e., what we do [here](https://github.com/DataBiosphere/hca-ingest/blob/d68a1b53219f84b69a60b75c3aa26ce4ad5d7de6/orchestration/hca_orchestration/solids/validate_egress.py#L40-L55)